app/src/main/java/ch/epfl/sdp/drone3d/map/offline/OfflineMapSaver.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.map.offline
+
+import androidx.lifecycle.LiveData
+import com.mapbox.mapboxsdk.geometry.LatLngBounds
+import com.mapbox.mapboxsdk.offline.OfflineManager
+import com.mapbox.mapboxsdk.offline.OfflineRegion
+
+/**
+ * Stores the metadata associated to a downloaded region
+ */
+data class OfflineRegionMetadata(val name:String, val bounds:LatLngBounds, val tileCount:Long,
```

Comment:
Maybe I missed something, but why do you define OfflineRegionMetadata in this interface if you never use it here ?

Reply:
I was not sure about this one as it seemed odd to have it specific to the implementation if I want every implementations to provide this metadata